README.md

```diff
@@ -1,6 +1,6 @@
 # README: Moviegram
 
-## Random Password Generator
```

Comment:
I like how you got rid of "random password generator"  because it didnt make any sense with our project